clusterman/batch/cluster_metrics_collector.py

```diff
@@ -139,6 +141,10 @@ def run(self) -> None:
 
                 successful = self.write_all_metrics()
 
+                logger.info('Reloading watcher config files')
+                if self.reload_watchers():
+                    self.initialize_clusterman_metrics_client()
+
```

Comment:
I'm wondering if you need to set watchers on the configs for and also re-initialize the pool managers here, similar to what you did for the autoscaler. (Couple of lines earlier you can see the pool managers are loaded, and have their state reloaded too.